src/utils/github-api.ts

```diff
@@ -24,21 +24,22 @@ import { CheckSuiteStatus, Reason, Subject } from '../typesGithub';
 
 // prettier-ignore
```

Comment:
Why are we ignoring prettier here?

Reply:
🤷 pre-existing. Can do a sweep in a follow up PR

Reply:
pushed a new commit which only adds the new description... not sure how the others got reformatted